configure.ac

```diff
@@ -280,20 +279,23 @@ AS_IF([test "x$ax_cv_c_compiler_vendor" = "xgnu"], [
 dnl ====================================================================================
 dnl  Find known host OS.
 
-OS_SPECIFIC_INCLUDES=""
 os_is_win32=0
 
+AUDIO_CFLAGS=""
+AUDIO_LIBS=""
+
 AS_CASE([${host_os}],
-	[darwin* | rhapsody*], [
-		OS_SPECIFIC_INCLUDES="-fpascal-strings -I/Developer/Headers/FlatCarbon"
-		OS_SPECIFIC_LINKS="-framework CoreAudio"],
+	[darwin*], [dnl this requires 10.5+
+		AUDIO_LIBS="-framework CoreAudio"],
 	[mingw32*], [
-		OS_SPECIFIC_LINKS="-lwinmm"
+		AUDIO_LIBS="-lwinmm"
 		os_is_win32=1],
-	[
-		OS_SPECIFIC_INCLUDES=""
-		OS_SPECIFIC_LINKS=""
-	])
+	[linux*], [dnl have OSS as fallback if no ALSA
+		AS_IF([test "x$enable_alsa" = "xyes"],[
```

Comment:
last ultra-niggle: `AS_IF([test "x$enable_alsa" = "xyes"], [` (the space after the comma) for consistency

Reply:
Done